### PR TITLE
Correcting redirects and adding logout 

### PR DESCRIPTION
**Pull request by fangyu3**
Updating redirect after login

Adding frontend logout

Moving cookie creation to backend

### CODE REVIEW

File: client/src/components/CheckoutForm.js
Comment (fangyu3) [inline client/src/components/CheckoutForm.js:13]: @bugatti12345 Make sure to add this for all future fetch in order to send cookie to backend

File: server/api/google_login_handler.py
Comment (fangyu3) [inline server/api/google_login_handler.py:31]: @bugatti12345 cookie creation was moved to server side because we can't set HTTP_only cookie on the frontend. I found out this after trying to get cookie from frontend but failed. 

Here is more info:
https://stackoverflow.com/questions/14691654/set-a-cookie-to-httponly-via-javascript

File: client/src/components/Header.js
Comment (johnsciutto) [inline client/src/components/Header.js]: Remember to remove the alerts if they were for testing things out.

File: client/src/pages/SignUp.js
Comment (johnsciutto) [inline client/src/pages/SignUp.js:95]: Same as above.

File: server/api/logout_handler.py
Comment (johnsciutto) [inline server/api/logout_handler.py]: Remember to remove `print` statements if they were to test things out.

**Review by sina-jamshidi — APPROVED** [review]
Good to go once comments are addressed

File: client/src/components/CheckoutForm.js
Comment (JoshL579) [reply client/src/components/CheckoutForm.js:13]: Oh just saw that now. Thank you for reminding

File: client/src/components/Header.js
Comment (JoshL579) [inline client/src/components/Header.js]: I think there is still possibility of returning 40x status code (ex. BE server down or connection fail), and we might need to add an `else` at bottom to` throw Error` to handle it

**Review by JoshL579 — APPROVED** [review]

File: client/src/components/Header.js
Comment (fangyu3) [reply client/src/components/Header.js]: It should be removed after alert context branch is merged

File: client/src/components/Header.js
Comment (fangyu3) [reply client/src/components/Header.js]: okk added else after else if to throw error

File: server/config.py
Comment (fangyu3) [inline server/config.py:14]: @bugatti12345 Assigned token expiry to 30 minutes. Originally 1800 was assigning tokens that were already expired.

File: client/src/components/Header.js
Comment (fangyu3) [inline client/src/components/Header.js:83]: @bugatti12345 Added throw error that takes into account of other 4xx/5xx status errors

**Pull request merged**
